### ISSUE TITLE
4.4 - CNN Evaluation Notebook

### ISSUE THREAD

**Issue by A01799073** [A01799073/AguacAIte-Labs#17]
Summarize CNN training curves, performance, and error analysis.

- [x] Plot training history
- [x] Discuss over/underfitting

**Comment by A01799073** [comment]
<img width="695" height="935" alt="Image" src="https://github.com/user-attachments/assets/03a3e1ad-d3b6-4ef7-b034-b768a28a21d9" />

**Overfitting analysis:**
The CNN shows clear signs of overfitting. Training loss rapidly converges to near zero and training accuracy approaches 100%, while validation accuracy remains stagnant around 66%. This gap indicates that the model is memorizing the training data but failing to generalize to unseen samples. This behavior is expected given the limited dataset size and the absence of data augmentation or stronger regularization techniques.

**Underfitting analysis:**
There is no evidence of underfitting, as the model is able to fit the training data effectively and achieve high training performance.
